### PR TITLE
Upgraded ruby to 2.2.4 plus 5 gems.

### PR DESCRIPTION
**Pull request by jasnow**
* Upgraded ruby to 2.2.4.
 * Upgraded 5 gems - included new assets.rb initializer file.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ before_install:
 - gem --version
 language: ruby
 rvm:
-- 2.2.3
+- 2.2.4
 env:
   matrix:
   - RAILS=4.2.4
```
Comment (orendon) [inline .travis.yml]: can we use `4.2.5` here as well? so we match the rails version on Gemfile

Comment (jasnow) [inline .travis.yml]: done

**Comment by jasnow** [comment]
@orendon - Just restart current Travis build. Mine went green this AM.

**Comment by jasnow** [comment]
@orendon - Add 4.2.5 as requested.

**Pull request merged**